datacamp_python_project_spec_lefthandedness.ipynb

```diff
@@ -0,0 +1,255 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7ffbb88f7588>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# load the left-handedness data and create a scatter plot\n",
+    "\n",
+    "import pandas as pd\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "data_url = \"https://gist.githubusercontent.com/mbonsma/8da0990b71ba9a09f7de395574e54df1/raw/aec88b30af87fad8d45da7e774223f91dad09e88/lh_data.csv\"\n",
+    "lefthanded_data = pd.read_csv(data_url, sep = ',')\n",
+    "\n",
+    "# this makes the plot appear in the notebook\n",
+    "%matplotlib inline\n",
+    "\n",
+    "plt.figure()\n",
+    "ax = plt.gca()\n",
+    "ax.plot(lefthanded_data['Age'], lefthanded_data['Female'], label=\"Female\", marker = 'o')\n",
```

Comment:
If you use the data oriented interface (not sure if that is the official name), you can save some keystroke and the label will be added automatically: `ax.plot('Age', 'Female', data=lefthanded_data, marker='o')`. Student don't learn this I believe, but the result will be the same so you can use whichever style you prefer.

Reply:
It's called [labeled data](https://matplotlib.org/users/prev_whats_new/whats_new_1.5.html#working-with-labeled-data-like-pandas-dataframes).